lib/core/filtering/validation_job_runner.rb

```diff
@@ -27,7 +25,7 @@ def execute
         validation_result = @connector_class.validate_filtering(@connector_settings.filtering[:draft])
 
         # currently only used for connectors -> DEFAULT domain can be assumed (will be changed with the integration of crawler)
-        ElasticConnectorActions.update_filtering_validation(@connector_settings.id, { DEFAULT_DOMAIN => validation_result })
+        ElasticConnectorActions.update_filtering_validation(@connector_settings.id, { Core::Filtering::DEFAULT_DOMAIN => validation_result })
```

Comment:
I don't quite get it. You removed `DEFAULT_DOMAIN = 'DEFAULT'` in row 14 above, so where does it come from now? I don't see a `require` for `core/filtering`.